src/modules/75-cd/pages/get-started-with-cd/__tests__/GetStartedWithCDV2.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { render, fireEvent, act } from '@testing-library/react'
+import { TestWrapper } from '@common/utils/testUtils'
+import { CDOnboardingActions } from '@common/constants/TrackingConstants'
+import GetStartedWithCDV2 from '../GetStartedWithCDv2'
+const trackEventMock = jest.fn()
+const mockHistoryPush = jest.fn()
+jest.mock('@common/hooks/useTelemetry', () => ({
+  useTelemetry: () => ({ identifyUser: jest.fn(), trackEvent: trackEventMock })
+}))
+
+// eslint-disable-next-line jest-no-mock
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useHistory: () => ({
+    push: mockHistoryPush
+  })
+}))
+describe('GetStartedWithCDV2 Component', () => {
+  test('renders without crashing', () => {
```

Comment:
Remember to keep some line breaks for readability.
```suggestion

const trackEventMock = jest.fn()
const mockHistoryPush = jest.fn()

jest.mock('@common/hooks/useTelemetry', () => ({
  useTelemetry: () => ({ identifyUser: jest.fn(), trackEvent: trackEventMock })
}))

// eslint-disable-next-line jest-no-mock
jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useHistory: () => ({
    push: mockHistoryPush
  })
}))

describe('GetStartedWithCDV2 Component', () => {
  test('renders without crashing', () => {
```